### DISCUSSION TITLE
install add-on problem

### DISCUSSION THREAD

**Discussion by cotroneo79** [OSGeo/grass#2353] [General]
Dear all,
Hi all,
I would like install extension, but when install extension i show this message and dont install. 
![ins](https://user-images.githubusercontent.com/104670864/166095966-363f2c44-fb99-47a9-811d-e22a100c27f3.JPG)

**Comment by ninsbl** [comment]
Hi @cotroneo79 it seems you try to install the addon system-wide (you use the `-s`-flag). System-wide installation requires administrator privellges (or at least write access to the target directory), which you probably don't have with your current user out of the box. Please try `g.extension` without `-s`.

**Comment by sunmck** [comment]
Hi all,

I also have a problem installing add-ons in GRASS 8.2. I want to install i.sentinel toolbox with `g.extension extension=i.sentinel` but get the error message _ERROR: Cannot open URL: http://wingrass.fsv.cvut.cz/grass82/addons/grass-8.2.0/i.sentinel.zip_.

It is even not possible to only show all available add-ons via `g.extension -l`, although the specified file exists and internet connection is secured:
![image](https://user-images.githubusercontent.com/116874799/210754312-f3a7f1c4-0799-4799-ab20-9c84f70143b0.png)

Does anyone have an idea how to solve this issue?

**Reply by neteler** [reply]
> I also have a problem installing add-ons in GRASS 8.2. I want to install i.sentinel toolbox with `g.extension extension=i.sentinel` but get the error message _ERROR: Cannot open URL: http://wingrass.fsv.cvut.cz/grass82/addons/grass-8.2.0/i.sentinel.zip_.

If you click on the link you can see that the file exists.
Quite likely you are behind a firewall or a proxy is to be used at your end.

**Comment by dnewcomb** [comment]
That was an issue in 8.2.0 on windows. Fixed in 8.2.1 rc1 for windows

**Reply by sunmck** [reply]
How do I install 8.2.1?

**Reply by nilason** [reply]
https://grass.osgeo.org/grass82/binary/mswindows/native/WinGRASS-8.2.1RC1-1-Setup.exe
